### PR TITLE
Contact Page

### PR DESCRIPTION
**Pull request by Gregory711**
Added contact page accessible via sidebar with working user feedback form powered by Google App Script and tested with Playwright.

## Checklist for completing pull request:
- [x] Test functionality on Mobile, Web, and Desktop
- [x] Verify that any tests for new functionality are created and functional.
- [x] If needed, update the Readme

### CODE REVIEW

**Comment by Thomas-Gallant** [comment]
In certain resizes it is impossible to see the submit button
I guess we either need a scrollview or to improve the dynamic resizing so everything fits. I'm thinking a scrollview might be better
image

**Comment by Thomas-Gallant** [comment]
A small thing that is bugging me is that hovering mouse over disabled "submit feedback" button shows indication that it is clickable when it is not. I'll try to figure out what is causing this.

**Comment by Gregory711** [comment]
> In certain resizes it is impossible to see the submit button I guess we either need a scrollview or to improve the dynamic resizing so everything fits. I'm thinking a scrollview might be better image

resolved

File: app/Pages/ContactPage.tsx [hunk 1]
```diff
@@ -1,9 +1,61 @@
...
+  const navigation: any = useNavigation();
+  const size = useWindowDimensions();
+  const [value, setValue] = React.useState("");
+  const [text, setText] = React.useState("");
+  const [label, setLabel] = React.useState("0/1000");
+  const [disabled, setDisabled] = React.useState(true);
+  const [placeholder, setPlaceholder] = React.useState("");
+  const [thankYouVisible, setThankYouVisible] = React.useState(false);
+  const [buttonText, setButtonText] = React.useState("Submit Feedback*");
+  const [errorVisible, setErrorVisible] = React.useState(false);
```
Comment (Thomas-Gallant) [inline app/Pages/ContactPage.tsx:28]: I used to do this a lot, having individual useStates for each value. I think creating a state object makes things easier to understand / maintain. 
```typescript
interface contactPageState {
  value: string
  text: string
  label: string
  placeholder: string
  buttonText: string
  buttonDisabled: boolean
}
const [contactPage, setContactPage] = React.useState<contactPageState>({
  buttonDisabled: true,
  buttonText: "Submit Feedback*",
  label: "0/1000",
  text: "",
  placeholder: "",
  value: ""
});
```

For retrieving a state would use ```contactPage.value```
And for updating a state we can do:
```typescript
setContactPage({
  ...contactPage,
  value: value,
  disabled: false,
  placeholder: placeholder,
});
```

**Review by Thomas-Gallant — APPROVED** [review]

File: app/Pages/ContactPage.tsx [hunk 2]
```diff
@@ -20,8 +84,161 @@ const ContactPage = () => {
...
+            </View>
+          </Card>
+        </View>
+        <Alert
+          show={contactPage.thankYouVisible}
+          title="Thank You!"
+          message={
+            `Your feedback has been submitted.\n\n` +
+            `We appreciate the time you took to help us improve our app.\n\n` +
+            `Your feeback will be taken into consideration.`
```
Comment (Thomas-Gallant) [inline app/Pages/ContactPage.tsx:190]: feeback  --> feedback

**Review by Thomas-Gallant — CHANGES_REQUESTED** [review]
After submitting feedback and navigating back home automatically, sidebar still shows as being on the contact page:
image

**Review by Thomas-Gallant — DISMISSED** [review]

**Review by Thomas-Gallant — APPROVED** [review]

**Pull request merged**